worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BlockTypesCache.java

```diff
@@ -245,9 +244,30 @@ private int parseProperties(String properties, Map<String, AbstractProperty<?>>
                     // Note: Throws IndexOutOfBoundsError if nothing is registered and blocksMap is empty
                     values[internalId] = type;
                 }
+            }*/
+
+            // Register "Reserved". Ensure air/reserved and 0/1/2/3
+            {
+                int internalId = 0;
+                for (String id : new String[]{"minecraft:__reserved__", "minecraft:air", "minecraft:cave_air",
+                        "minecraft:void_air"}) {
+                    String defaultState = blockMap.remove(id);
+                    if (defaultState == null) {
+                        defaultState = id;
+                    }
+                    if (values[internalId] != null) {
+                        throw new IllegalStateException(
+                                "Invalid duplicate id for __reserved__! Something has gone very wrong. Are " +
```

Comment:
Is it likely that this happens? We could wrap that into a try/catch and fire the exception to obtain the stacktrace to see _if_ it's shaded